app/assets/javascripts/discourse/app/controllers/delete-topic-confirm.js

```diff
@@ -0,0 +1,38 @@
+import I18n from "I18n";
+import discourseComputed from "discourse-common/utils/decorators";
+import { alias } from "@ember/object/computed";
+import { inject } from "@ember/controller";
+import Controller from "@ember/controller";
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+
+// Modal that displays confirmation text when user deletes a topic
+// The modal will display only if the topic exceeds a certain amount of views
+export default Controller.extend(ModalFunctionality, {
+  topicController: inject("topic"),
+  minViews: alias("siteSettings.min_topic_view_amount_for_delete_confirmation"),
```

Comment:
You probably don't need an alias here. In the template you can use the full key path and it'll work.